tests/cache/tests.py

```diff
@@ -1221,6 +1221,12 @@ def test_creates_cache_dir_if_nonexistent(self):
         cache.set('foo', 'bar')
         os.path.exists(self.dirname)
 
+    def test_cache_write_unpickable_type(self):
+        try:
```

Comment:
It would probably be preferred to simply set/get the value instead of using try/except. See 071801ccff970682a799ce754431a3c3ce3d6902.